nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -308,6 +308,10 @@ public List<Namespace> listNamespaces(Namespace namespace) throws NoSuchNamespac
   }
 
   public boolean dropNamespace(Namespace namespace) throws NamespaceNotEmptyException {
+    if (namespace.isEmpty()) {
+      // As creating empty namespaces is not supported
+      throw new NoSuchNamespaceException("Namespace does not exist: %s", namespace);
```

Comment:
I'd wrap the namespace name in single quotes, as it is done in other places; it gives more understandable messages for empty namespaces.

Reply:
Many existing places we don't wrap it. Like `loadNamespaceMetadata`, `updateProperties`. 

Also checked that other catalogs doesn't wrap it too. 
So, some common test may fail if I wrap it. 
We can analyze in the follow up.

Reply:
maybe just define a `checkNamespaceIsValid` method that is called in all places
```
private void checkNamespaceIsValid(Namespace namespace) {
    if (namespace.isEmpty()) {
      throw new NoSuchNamespaceException("Invalid namespace: %s", namespace);
    }
  }
```
The same is being done in `RESTSessionCatalog`. Also I agree, we don't wrap the namespace in single quotes in other places, so this one should also not wrap

Reply:
done